ann_benchmarks/algorithms/tiledb/module.py

```diff
@@ -47,15 +66,18 @@ def fit(self, X):
         elif X.dtype == "float32":
```

Comment:
We can update this to use the numpy arrays directly for ingestion

Reply:
Thanks, added this as a TODO and will fix this the next time I set up an EC2 instance and run this. Doing this b/c it seems nice to get this PR merged versus leaving in flux until then.